Makefile

```diff
@@ -60,13 +65,18 @@ lint: ## Run linter
 	@echo "==> Linting all packages..."
 	golangci-lint run --timeout 5m
 
+.PHONY: scan
+scan: ## Run static code analysis
+	@echo "==> Scanning the code..."
+	gokart scan . -v -g
+
 .PHONY: fix-lint
 fix-lint: ## Fix linting errors
 	@echo "==> Fixing lint errors"
 	golangci-lint run --fix --timeout 5m
 
 .PHONY: check
-check: test fix-lint ## Run tests and linters
+check: scan test fix-lint ## Run tests and linters
```

Comment:
run the scanner

Reply:
let's please flow a red -> green -> refactor flow, meaning tests before linting